ebs/ebs.go

```diff
@@ -281,9 +297,9 @@ func (d *Driver) Info() (map[string]string, error) {
 	infos["DefaultKmsKey"] = d.DefaultKmsKeyID
 	infos["DefaultFSType"] = d.DefaultFSType
 	infos["DefaultEncrypted"] = fmt.Sprint(d.DefaultEncrypted)
-	infos["InstanceID"] = d.ebsService.InstanceID
-	infos["Region"] = d.ebsService.Region
-	infos["AvailiablityZone"] = d.ebsService.AvailabilityZone
+	infos["InstanceID"] = d.ebsService.GetInstanceID()
```

Comment:
old argument here about the use of getters, but not really a necessary code fix.